packages/RichTextEditing-Core.package/DocumentReadStream.class/instance/lineBreak.st

```diff
@@ -0,0 +1,6 @@
+stop conditions
+lineBreak
+
+	^ ((self collection at: self position) = Character lf)
```

Comment:
Create a method for this (`currentCharacterIs: aCharacter`).

Reply:
What do you think about the method ```currentCharacter``` to make it more flexible?